serving/pom.xml

```diff
@@ -278,12 +278,56 @@
       <artifactId>embedded-redis</artifactId>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>jakarta.validation</groupId>
       <artifactId>jakarta.validation-api</artifactId>
       <version>${jakarta.validation.api.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
+      <artifactId>spring-security-core</artifactId>
+      <version>5.3.0.RELEASE</version>
```

Comment:
These versions need to be updated, and ideally we should set versions in the root pom.xml.